src/test/java/seedu/address/testutil/TypicalOrders.java

```diff
@@ -17,11 +17,15 @@
 public class TypicalOrders {
 
     public static final Order SALESORDER1 = new OrderBuilder().withCustomer("Josh")
-            .withDate("2021-09-18").withAmount("10").withLabel("testorder1").withId(1).build();
+            .withDate("2021-09-18").withAmount("10").withLabel("testorder1").withIsComplete(true).build();
     public static final Order SALESORDER2 = new OrderBuilder().withCustomer("Mac")
-            .withDate("2021-09-19").withAmount("15").withIsComplete(true).withId(2).withLabel("testorder2").build();
+            .withDate("2021-09-19").withAmount("20").withIsComplete(true).withLabel("testorder2").build();
     public static final Order SALESORDER3 = new OrderBuilder().withCustomer("Clark")
-            .withDate("2021-09-20").withAmount("20").withIsComplete(true).withId(3).withLabel("testorder3").build();
+            .withDate("2021-09-20").withAmount("15").withIsComplete(true).withLabel("testorder3").build();
+    public static final Order SALESORDER4 = new OrderBuilder().withCustomer("Justin").withLabel("testorder4")
+            .withDate("2021-09-20").withAmount("20").build();
+    public static final Order SALESORDER5 = new OrderBuilder().withCustomer("Stuart").withLabel("testorder5")
+            .withDate("2021-09-20").withAmount("5").withIsComplete(false).build();
```

Comment:
What's the reason you removed `withId` method here?

Reply:
I changed the OrderBuilder to use the regular Order counter instead of having a default order id. So the Orders still end up having the ids specified before. But I can revert it if this doesn't make sense for testing

Reply:
@Tanishq4331 I believe the test case will pass if you add `withid` here.

Reply:
Yep, this is what I ended up doing. Thanks!